Examples/Scripts/Python/gaussian_sum_filter.py

```diff
@@ -0,0 +1,168 @@
+            RootParticleReader(
+                level=acts.logging.INFO,
+                filePath=str(inputParticlePath.resolve()),
+                particleCollection="particles_input",
+                orderedEvents=False,
+            )
+        )
+
+    s = addFatras(
+        s,
+        trackingGeometry,
+        field,
+        rnd=rnd,
+    )
+
+    s = addDigitization(
+        s,
+        trackingGeometry,
+        field,
+        digiConfigFile=digiConfigFile,
+        rnd=rnd,
+    )
+
+    selAlg = acts.examples.TruthSeedSelector(
+        level=acts.logging.INFO,
+        ptMin=500 * u.MeV,
+        nHitsMin=9,
+        inputParticles="particles_initial",  # fatrasCfg.outputParticlesInitial,
+        inputMeasurementParticlesMap="measurement_particles_map",  # digiCfg.outputMeasurementParticlesMap,
+        outputParticles="particles_seed_selected",
+    )
+    s.addAlgorithm(selAlg)
+
+    inputParticles = selAlg.config.outputParticles
+
+    smearAlg = acts.examples.ParticleSmearing(
+        level=acts.logging.INFO,
+        inputParticles=inputParticles,
+        outputTrackParameters="smearedparameters",
+        randomNumbers=rnd,
+        # Gaussian sigmas to smear particle parameters
+        # sigmaD0=20 * u.um,
+        # sigmaD0PtA=30 * u.um,
+        # sigmaD0PtB=0.3 / u.GeV,
+        # sigmaZ0=20 * u.um,
+        # sigmaZ0PtA=30 * u.um,
+        # sigmaZ0PtB=0.3 / u.GeV,
+        # sigmaPhi=1 * u.degree,
+        # sigmaTheta=1 * u.degree,
+        # sigmaPRel=0.01,
+        # sigmaT0=1 * u.ns,
+        # initialVarInflation=[1, 1, 1, 1, 1, 1],
+    )
+    s.addAlgorithm(smearAlg)
+
+    truthTrkFndAlg = acts.examples.TruthTrackFinder(
+        level=acts.logging.INFO,
+        inputParticles=inputParticles,
+        inputMeasurementParticlesMap="measurement_particles_map",  # digiCfg.outputMeasurementParticlesMap,
+        outputProtoTracks="prototracks",
+    )
+    s.addAlgorithm(truthTrkFndAlg)
+
+    gsfOptions = {
+        "maxComponents": 12,
+        "abortOnError": False,
+        "disableAllMaterialHandling": False,
+    }
+
+    gsfAlg = acts.examples.TrackFittingAlgorithm(
+        level=acts.logging.INFO,
+        inputMeasurements="measurements",  # digiCfg.outputMeasurements,
+        inputSourceLinks="sourcelinks",  # digiCfg.outputSourceLinks,
+        inputProtoTracks=truthTrkFndAlg.config.outputProtoTracks,
+        inputInitialTrackParameters=smearAlg.config.outputTrackParameters,
+        outputTrajectories="gsf-trajectories",
+        directNavigation=False,
+        pickTrack=-1,
+        trackingGeometry=trackingGeometry,
+        fit=acts.examples.TrackFittingAlgorithm.makeGsfFitterFunction(
+            trackingGeometry, field, **gsfOptions
+        ),
+    )
+
+    s.addAlgorithm(gsfAlg)
+
+    return s
+
+
+if "__main__" == __name__:
+    srcdir = Path(__file__).resolve().parent.parent.parent.parent
+
+    detector, trackingGeometry, decorators = GenericDetector.create()
```

Comment:
Have you tested this on ODD at this point? (GenericDetector as default here is fine though)

Reply:
With the binaries, yes. With the python version not yet...